### PR TITLE
Travis CI Initialization & automated integration test for azure-simple

### LINKED ISSUE #8: Build test infrastructure
We should be able to validate that the project works end to end before and after each commit with a CI system.

Some ideas on how to test a Terraform deployment here:  https://devops.stackexchange.com/questions/863/how-to-test-a-terraform-configuration

### PR DESCRIPTION
**Pull request by NathanielRose**
Closes #8
Allows automated builds and flux service confirmation with cluster deployment on PRs.
Uses hard-coded output path for Kube_config and terraform deployment in Go test.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+go:
+- 1.11.x
+- master
+
+go_import_path: github.com/NathanielRose/bedrock
+
+before_install:
+- echo $clientID
+- echo $clientSecret
```
Comment (dtzar) [inline .travis.yml]: let's get rid of these echos ;)

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- echo $publicKey
+- echo publicKey
+- wget https://releases.hashicorp.com/terraform/0.11.11/terraform_0.11.11_linux_amd64.zip
```
Comment (dtzar) [inline .travis.yml]: Let's put this into a ENV_VAR so it's easy to update the version without having to modify the travis file

File: README.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # bedrock
 
+[![Build Status](https://travis-ci.org/NathanielRose/bedrock.svg?branch=master)](https://travis-ci.org/NathanielRose/bedrock)
```
Comment (dtzar) [inline README.md]: Needs to point to master branch of main repo

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+- curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+- echo "deb https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee -a /etc/apt/sources.list.d/kubernetes.list
+- sudo apt-get update
+- sudo apt-get install -y kubectl
+- sudo apt-get install azure-cli
+- curl https://raw.githubusercontent.com/helm/helm/master/scripts/get > get_helm.sh
+- chmod 700 get_helm.sh
+- ./get_helm.sh
+- az -v
+- terraform -v
```
Comment (dtzar) [inline .travis.yml]: Remove since we explicitly install a version.

**Comment by NathanielRose** [comment]
Cleaning up PR and consolidating commits to a branch pr.

**Comment by andrebriggs** [comment]
@NathanielRose From the docs this [line](https://github.com/Microsoft/bedrock/blob/2984b7ad91861a5dca5ee88f2a52fc01355769af/testing/bedrock_azure_simple_test.go#L52) just writes to STDOUT. Will we assert that flux namespace exists?

**Pull request closed without merging**